gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/store/hdfs/SimpleHDFSConfigStoreFactory.java

```diff
@@ -38,6 +41,47 @@
   protected static final String SIMPLE_HDFS_SCHEME_PREFIX = "simple-";
   protected static final String HDFS_SCHEME_NAME = "hdfs";
 
+  public final static String DEFAULT_STORE_URI_KEY = "default_store_uri";
+
+  public SimpleHDFSConfigStoreFactory() {
+    this(ConfigFactory.empty());
+  }
+
+  private final Optional<URI> defaultStoreURI;
+  private final FileSystem defaultStoreFS;
+
+  public SimpleHDFSConfigStoreFactory(Config factoryConfig) {
```

Comment:
Good point. I have modified the default constructor to use systemProperties/resources to configure a global instance. For other instances, we can register specific config using ConfigStoreFactoryRegister.register(). 

Actually, there needs to be a follow-up to modify the scheme handling as currently we can have only one HDFS-based instance.